modelexpress_client/python/modelexpress/metadata/publisher.py

```
@@ -133,15 +136,56 @@ def stop(self) -> None:
+        clean shutdown with a crash.
+        """
+        if self._nixl_manager is None:
+            return
+        try:
+            disconnected = self._nixl_manager.disconnect_remote_agents()
+            if disconnected:
+                logger.info(
+                    f"[Worker {self._worker_rank}] Disconnected "
+                    f"{disconnected} NIXL peer(s) on shutdown"
+                )
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)